src/js/containers/covid19/budgetCategories/BudgetCategoriesTableContainer.jsx

```diff
@@ -0,0 +1,193 @@
+/**
+ * BudgetCategoriesTableContainer.jsx
+ * Created by James Lee 6/5/20
+ */
+
+import React, { useState, useEffect, useCallback } from 'react';
+import PropTypes from 'prop-types';
+import { Table, Pagination, TooltipWrapper, Picker } from 'data-transparency-ui';
```

Comment:
👀 

Nice

Reply:
🙌 putting the component library to good use!